NetKAN/WarrigalsAutopilot.netkan

```diff
@@ -0,0 +1,10 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "WarrigalsAutopilot",
+    "name"         : "Warrigal's Autopilot",
+    "author"       : "Warrigal",
+    "$kref"        : "#/ckan/github/tswett/WarrigalsAutopilot",
+    "license"      : "GPL-3.0",
+    "version"      : "0.0.1",
```

Comment:
The version will be retrieved from the releases on GitHub on the fly, so it would be easier for maintenance if this line was removed.